storage_devices/disk.py

```diff
@@ -311,4 +324,25 @@ def get_sysfs_addr(cls, device_id):
 
         return sysfs_addr.full_path
 
+    @classmethod
+    def identify(cls, device_name: str) -> bool:
+        output = TestRun.executor.run(
+            f"realpath /sys/block/{device_name}/device/driver | grep virtio"
+        )
+        return output.exit_code == 0
+
 
+class DiskRegistry:
+    driver_types = [NvmeDisk, SataDisk, VirtioDisk]
+
+    @classmethod
+    def get_disk_type(cls, disk_path: str):
+        device_name = TestRun.executor.run(f"realpath {disk_path}").stdout.split("/")[2]
```

Comment:
`identify()` should receive the path as raw as possible. All processing should be done in `identify()` itself